app/views/main.scala.html

```diff
@@ -40,7 +40,8 @@
   gaCustomTitle: Option[String] = None,
   optimizelyAudience: Option[String] = None,
   pageHeader: Option[Html] = None,
-  script: Option[Html] = None
+  script: Option[Html] = None,
+  isAuthenticated: Boolean = true
```

Comment:
defaulting isAuthenticated is confusing as it suggests every page in tai-frontend is authenticated. We should be asking auth if it's authenticated and not setting it ourselves.